src/browser/__tests__/ReactDOMSVG-test.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright 2013-2015, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @emails react-core
+ */
+
+/*jslint evil: true */
+
+'use strict';
+
+var React = require('React');
+var ReactTestUtils = require('ReactTestUtils');
+
+describe('ReactDOMSVG', function() {
+  it("allows a SVG element", function() {
+    var element = React.createElement('svg', {xmlnsXlink: 'http://www.w3.org/1999/xlink'});
```

Comment:
Would be nice if people didn't need to specify this by hand. I guess you need to when writing svg outside of React though?